src/test/java/org/mskcc/oncokb/curation/web/rest/FeatureFlagResourceIT.java

```diff
@@ -0,0 +1,766 @@
+        defaultFeatureFlagShouldBeFound("enabled.equals=" + DEFAULT_ENABLED);
+
+        // Get all the featureFlagList where enabled equals to UPDATED_ENABLED
+        defaultFeatureFlagShouldNotBeFound("enabled.equals=" + UPDATED_ENABLED);
+    }
+
+    @Test
+    @Transactional
+    void getAllFeatureFlagsByEnabledIsNotEqualToSomething() throws Exception {
+        // Initialize the database
+        featureFlagRepository.saveAndFlush(featureFlag);
+
+        // Get all the featureFlagList where enabled not equals to DEFAULT_ENABLED
+        defaultFeatureFlagShouldNotBeFound("enabled.notEquals=" + DEFAULT_ENABLED);
+
+        // Get all the featureFlagList where enabled not equals to UPDATED_ENABLED
+        defaultFeatureFlagShouldBeFound("enabled.notEquals=" + UPDATED_ENABLED);
+    }
+
+    @Test
+    @Transactional
+    void getAllFeatureFlagsByEnabledIsInShouldWork() throws Exception {
+        // Initialize the database
+        featureFlagRepository.saveAndFlush(featureFlag);
+
+        // Get all the featureFlagList where enabled in DEFAULT_ENABLED or UPDATED_ENABLED
+        defaultFeatureFlagShouldBeFound("enabled.in=" + DEFAULT_ENABLED + "," + UPDATED_ENABLED);
+
+        // Get all the featureFlagList where enabled equals to UPDATED_ENABLED
+        defaultFeatureFlagShouldNotBeFound("enabled.in=" + UPDATED_ENABLED);
+    }
+
+    @Test
+    @Transactional
+    void getAllFeatureFlagsByEnabledIsNullOrNotNull() throws Exception {
+        // Initialize the database
+        featureFlagRepository.saveAndFlush(featureFlag);
+
+        // Get all the featureFlagList where enabled is not null
+        defaultFeatureFlagShouldBeFound("enabled.specified=true");
+
+        // Get all the featureFlagList where enabled is null
+        defaultFeatureFlagShouldNotBeFound("enabled.specified=false");
+    }
+
+    /**
+     * No UserResourceIT, so comment it first
+     */
+    // @Test
+    // @Transactional
+    // void getAllFeatureFlagsByUserIsEqualToSomething() throws Exception {
+    //     // Initialize the database
+    //     featureFlagRepository.saveAndFlush(featureFlag);
+    //     User user;
+    //     if (TestUtil.findAll(em, User.class).isEmpty()) {
+    //         user = UserResourceIT.createEntity(em);
+    //         em.persist(user);
+    //         em.flush();
+    //     } else {
+    //         user = TestUtil.findAll(em, User.class).get(0);
+    //     }
+    //     em.persist(user);
+    //     em.flush();
+    //     featureFlag.addUser(user);
+    //     featureFlagRepository.saveAndFlush(featureFlag);
+    //     Long userId = user.getId();
+
+    //     // Get all the featureFlagList where user equals to userId
+    //     defaultFeatureFlagShouldBeFound("userId.equals=" + userId);
+
+    //     // Get all the featureFlagList where user equals to "invalid-id"
+    //     defaultFeatureFlagShouldNotBeFound("userId.equals=" + "invalid-id");
+    // }
```

Comment:
Was it not possible to get this test to pass?

Reply:
The main problem is that we do not have the UserResourceIT class, since we do not need to use this function now, I have just committed it.